aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -60,6 +70,27 @@ void convert_indices_from_coo_to_csr_cpu(const Tensor& result, const Tensor& inp
     data_out[i] = static_cast<output_t>(numel);
 }
 
+template <typename input_t, typename output_t>
+void convert_indices_from_csr_to_coo_cpu(const Tensor& result, const Tensor& crow_indices, const Tensor& col_indices) {
+  int64_t nrows = crow_indices.numel() - 1;
+  if (nrows == 0) {
+    result.zero_();
+    return;
+  }
+  auto crow_indices_ = crow_indices.expect_contiguous();
+  const input_t* crow_indices_data_in = crow_indices_->data_ptr<input_t>();
+  TORCH_INTERNAL_ASSERT(result.is_contiguous());
+  output_t* data_out = result.data_ptr<output_t>();
+
+  result.copy_(*col_indices.expect_contiguous());
```

Comment:
This does not sound right: the op involves broadcasting and does unnecessary copy. Recall, the shape of `result` is `(2, numel)` while the shape of `col_indices` is `(numel,)`. The in-place copy should be applied to the second row of `result` only.

Reply:
Nit: It would be nice to rename `result` to `indices` for better code readability.

Reply:
Changed to `indices.select(0, 1).copy_(...);`.

Reply:
Or would you prefer another approach?